lib/evmone/execution_state.hpp

```diff
@@ -232,25 +232,5 @@ class ExecutionState
             m_tx = host.get_tx_context();
         return m_tx;
     }
-
-    /// Get initcode by its hash from transaction initcodes.
-    ///
-    /// Returns empty bytes_view if no such initcode was found.
-    [[nodiscard]] bytes_view get_tx_initcode_by_hash(const evmc_bytes32& hash) noexcept
-    {
-        if (!m_initcodes.has_value())
-        {
-            m_initcodes.emplace();
```

Comment:
I think `m_initcodes` is not needed any more and this may propagate to transaction definition and JSON loading.